Tools/cases_generator/analysis.py

```diff
@@ -171,8 +171,6 @@ def parse_file(self, filename: str, instrs_idx: dict[str, int]) -> None:
                 case parsing.Pseudo(name):
                     self.pseudos[name] = thing
                     self.everything.append(thing)
-                case _:
-                    typing.assert_never(thing)
```

Comment:
FYI

````
analysis.py:175: error: Statement is unreachable  [unreachable]
                        typing.assert_never(thing)
````

Reply:
Huh, I thought that was the whole point of `assert_never` -- to have the type checker assert that that branch of code was unreachable. That seems like a bug in mypy if it's flagging these intentionally unreachable branches as unreachable :/

Reply:
Even mypy 1.5.1 has the same issue. I think that it's worth reporting to the mypy team.

Reply:
Not sure where my comment on this went, but I strongly prefer to keep those two lines (they exist specifically so a type checker can warn if you are missing a case). You could add
```
# type: ignore # mypy complains about this incorrectly
```
to the end of the line where the error occurs. Or just set `warn_unreachable = False` in `mypy.ini`.